app/views/content/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -56,15 +56,22 @@ keywords:
   - Settlement Scheme
   - Statement
   - Comparability
+  - Overseas degree equivalency
 ---
 
-As an international candidate you’ll be welcomed by many teacher training providers in England.
 
-Train in England and you'll graduate from a world-class institution, with qualifications recognised around the world.
 
-Not looking to train in England? Consider training in [Wales](https://educators.wales/home), [Scotland](https://teachinscotland.scot/) or [Northern Ireland](https://www.education-ni.gov.uk/articles/initial-teacher-education-courses-northern-ireland).
 
-### What you need to start training
+
+Teacher training in England is practical, hands-on and supportive. You’ll have the opportunity to train at a world class institution where creativity and innovation is valued, gaining lots of classroom experience along the way. You’ll leave with qualified teacher status (QTS), a professional qualification valued globally.
+
+You can apply to train to teach in England from any country in the world.
+
+You can also apply to train to teach in [Wales](https://educators.wales/home), [Scotland](https://teachinscotland.scot/) or [Northern Ireland](https://www.education-ni.gov.uk/articles/initial-teacher-education-courses-northern-ireland).
+
+
+
+### 1. Check your qualifications
```

Comment:
I think these steps should really be h2s instead of h3s, as otherwise there is a skipped heading level

Any reason not to change to h2s @peteryates ?